tests/periph_i2c/tests/periph_i2c_if.py

```diff
@@ -22,13 +22,6 @@ class PeriphI2cIf(DutShell):
     DEFAULT_LEN = 10
     DEFAULT_DATA = [0, 1, 2]
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-    def is_connected_to_board(self):
```

Comment:
The original use for this was for the autoconnect feature.  This needed implementation on the application level.  I guess since autoconnect was dropped we can drop it.

Reply:
the `__init__` is not needed, bc super is called by default if none more specific exists.

the `is_connected_to_board` does not work anymore bc the `i2c_id` function does not exist.

That's why I removed both